module/zfs/metaslab.c

```diff
@@ -2934,6 +2934,30 @@ metaslab_sync(metaslab_t *msp, uint64_t txg)
 	dmu_tx_commit(tx);
 }
 
+void
+metaslab_potentially_unload(metaslab_t *msp, uint64_t txg)
+{
+	/*
+	 * If the metaslab is loaded and we've not tried to load or allocate
+	 * from it in 'metaslab_unload_delay' txgs, then unload it.
+	 */
+	if (msp->ms_loaded &&
+	    msp->ms_disabled == 0 &&
+	    msp->ms_selected_txg + metaslab_unload_delay < txg) {
+		for (int t = 1; t < TXG_CONCURRENT_STATES; t++) {
```

Comment:
nit: I think this is the first use of "int t" in a for loop. Elsewhere in the code the int is usually 'i' (or a few 'c' or 'd')

Reply:
This was just a straight relocation of the existing code; I could change it if you'd like, but I think we use t elsewhere to iterate over txgs.

Reply:
my code search doesn't show any cases of `for (int t` but this is a nit

Reply:
Usually we don't declare `int t` in the loop, but `for (t =` appears in the code a couple dozen times.